.github/workflows/test-fast.yml

```diff
@@ -0,0 +1,26 @@
+name: Test
+
+on:
+  push:
+  workflow_dispatch:
```

Comment:
I'll add `pull_request` here and will [skip duplicate jobs](https://github.community/t/duplicate-checks-on-push-and-pull-request-simultaneous-event/18012/6#M9370). So, external PRs will not be a kind on second class citizens.